x-pack/legacy/plugins/lens/public/indexpattern_plugin/plugin.tsx

```diff
@@ -48,9 +39,7 @@ const plugin = new IndexPatternDatasourcePlugin();
 export const indexPatternDatasourceSetup = () => {
   plugin.setup(npSetup.core, {
     chrome,
-    interpreter: {
-      functionsRegistry,
-    },
+    expressions: npSetup.plugins.expressions,
```

Comment:
I believe you need to change our mocks in the Lens tests, since you've renamed the dependency

Reply:
tests seem to work